specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -13750,7 +13769,8 @@
         },
         "affinityId": {
           "type": "string",
-          "title": "An identifier which can be passed when adding a task to request that the task be scheduled close to this compute node."
+          "title": "An identifier which can be passed when adding a task to request that the task be scheduled on this node.",
+          "description": "Note that this is just a soft affinity. If the target node is busy or unavailable at the time the task is scheduled, then the task will be scheduled elsewhere."
```

Comment:
>Note [](start = 26, length = 4)

this was in a title somewhere else - I think it is probably better in the description